pom.xml

```diff
@@ -9,6 +9,13 @@
     <packaging>pom</packaging>
     <version>0.0.1-SNAPSHOT</version>
 
+    <scm>
+        <connection>scm:git:https://github.com/SmarterApp/TDS_ConfigService.git</connection>
+        <developerConnection>scm:git:https://git@github.com/SmarterApp/TDS_ConfigService.git</developerConnection>
+        <url>https://github.com/SmarterApp/TDS_ConfigService</url>
+        <tag>HEAD</tag>
+    </scm>
```

Comment:
Configured maven with the SCM values that allow the maven-release-plugin to push version bump commits.